articles/node-eslint/index.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: node-eslint
+title: Linting in Node using ESLint
+description: Introduction to Linters and a quick demo of how to set up ESLint with NodeJS.
+author: saiharsha-balasubramaniam
+date: 2020-08-12T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/node-eslint/hero.jpg
+    alt: linters
+---
+
+A linter is a computer program that analyzes and checks source code. It flags programming errors, indentation errors, formatting errors, bugs, and suspicious constructs.
+
+<!--more-->
+
+### Table of Contents
+
+- [Introducting to Linting](#introduction-to-linting)
+- [Advantages of a Linter](#advantages-of-a-linter)
+- [ESLint and Node](#eslint-and-node)
+- [Airbnb Style Guide](#airbnb-style-guide)
+- [Further Reading](#further-reading)
+
+### Introduction to Linting
+
+Linters are tools that perform **static code analysis** in software testing.
+
+Static Code Analysis is the analysis of computer software that is performed on static code. It is performed by looking at source code without running it. This process is usually done using a software or an automated tool. Static Code Analysis is done as a part of **Continuous Integration and Delivery**.
+
```

Comment:
Static analysis is performed during the coding phase - this is before CI/CD is performed.